tests/test_macaroons.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+from webapp.app import app
+from webapp.macaroons import MacaroonRequest
+
+
+class TestMacaroons(unittest.TestCase):
+    def setUp(self):
+        """
+        Set up Flask app for testing
+        """
+        app.testing = True
+        self.caveat_id = "caveat_id"
+        self.request_result = {"caveat_id": self.caveat_id}
+        self.discharge = "my_discharge"
+        self.response_result = {"discharge": self.discharge}
+
+    def test_request(self):
+        """
+    Should return always the same structure
+    """
+
+        openid_macaroon = MacaroonRequest(caveat_id=self.caveat_id)
+        self.assertEqual(
+            openid_macaroon.getExtensionArgs(), self.request_result
```

Comment:
Here are you simply testing internal logic of the `MacaroonRequest` library? We shouldn't test other people's code directly - that code should be tested by them, but even if it hasn't we should test it works through testing our usage of their module, rather than the module itself.

Again, I'd suggest removing this test.

Reply:
Well, it's not really a library, it's just a module I took from snapcraft. It was not unit tested on that project.